src/main/java/thePackmaster/cards/farmerpack/SowTheSeeds.java

```diff
@@ -0,0 +1,65 @@
+package thePackmaster.cards.farmerpack;
+
+import com.megacrit.cardcrawl.actions.common.MakeTempCardInHandAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import thePackmaster.actions.pixiepack.DrawSpecificCardAction;
+
+import java.util.Iterator;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+import static thePackmaster.util.Wiz.atb;
+import static thePackmaster.util.Wiz.att;
+
+public class SowTheSeeds extends AbstractFarmerCard {
+    public final static String ID = makeID("SowTheSeeds");
+    private boolean attack = false;
+    private boolean skill = false;
+    private boolean status = false;
+    private boolean power = false;
+    private boolean curse = false;
+    private int count = 0;
+    public SowTheSeeds() {
+        super(ID, 1, CardType.SKILL, CardRarity.UNCOMMON, CardTarget.SELF);
+        baseMagicNumber = magicNumber = 2;
+        cardsToPreview = new Fertilizer();
+    }
+
+    public void use(AbstractPlayer p, AbstractMonster m) {
+        atb(new MakeTempCardInHandAction(new Fertilizer()));
+        Iterator<AbstractCard> typeDetect = AbstractDungeon.player.drawPile.group.iterator();

```

Comment:
This code will always draw attacks and skills if there are any of them in your draw pile, and it always draw the _first_ card of the type in the draw pile. That's not how a player would expect an effect like this to work.

To be consistent with other effects like this, I'd suggest an approach like the following:
* Make a new list with all cards in the draw pile
* Select a random card from that list
* Add an action to draw that card
* Remove all cards of the same type as that card from the list
* Repeat until there are no cards left in the list or the right number of cards have been drawn

If it helps, BoosterTutor and BoosterTutorAction do something similar to this, just looking for cards based on different packs instead of different card types. It's a good example to follow

There's also several other things that you should fix about this:
* Use local variables instead of fields
* If you need a loop, use a for loop instead of iterators (e.g. `for (AbstractCard i : AbstractDungeon.player.drawPile) { ... }`)
* Put this all inside an action (you can make it a separate class like how BoosterTutorAction is used by BoosterTutor)